examples/default/main.tf

```diff
@@ -1,5 +1,34 @@
-provider "lacework" {}
+# This example assumes that Lacework CLI has been set up.  
+# Instructions for Lacework CLI setup - installation and configuration sections here: https://www.lacework.com/blog/running-with-lacework-cli/
 
-module "lacework_module" {
-  source = "../.."
+# This example assumes that you have multiple AWS profiles in your ~/.aws/credentials file that map to your audit and log_archive AWS accounts
+
+provider "aws" {
+  alias  = "log_archive"
+  profile  = "<profile name for log_archive account in ~/.aws/credentials>"
+  # e.g. profile  = "287453222145_AWSAdministratorAccess"
+  region = "us-east-1"
+}
+
+provider "aws" {
+  alias  = "audit"
+  profile  = "<profile name for audit account in ~/.aws/credentials>"
+  # e.g. profile  = "918733600796_AWSAdministratorAccess"
+  region = "us-east-1"
 }
+
+module "control_tower_integration_setup" {
+  source  = "lacework/cloudtrail-controltower/aws"
+  # source  = "./terraform-aws-cloudtrail-controltower"
+  version = "~> 0.1"
+  providers = {
+    aws.audit = aws.audit
+    aws.log_archive = aws.log_archive
+  }
+  aws_organization_id        = "<aws_org_id_e.g. o-XXXXXXXXXX>"
+  # SNS Topic ARN is usually in the form: arn:aws:sns:<control_tower_region>:<aws_audit_account_id>:aws-controltower-AllConfigNotifications
+  sns_topic_arn   = "arn:aws:sns:<control_tower_region>:<aws_audit_account_id>:aws-controltower-AllConfigNotifications"
+  # S3 Bucket ARN is usually in the form: arn:aws:s3:::aws-controltower-logs-<log_archive_account_id>-<control_tower_region>
+  s3_bucket_arn = "arn:aws:s3:::aws-controltower-logs-<log_archive_account_id>-<control_tower_region>"
+}
```

Comment:
Last thing, our principle for examples is that every example is a living example. Which means that
it has to be able to be executed by our CI tests and it doesn't look like this example is running
successfully in our pipeline.

![Screen Shot 2021-08-05 at 5 49 41 PM](https://user-images.githubusercontent.com/5712253/128431149-d23b9062-41a7-45c5-8996-aa029f1a997f.png)

Lets find some time to look at these examples together over Zoom

Reply:
Well, this is a bit of a circular reference problem since the example is calling this module which has not been published yet.  That is the line that is erroring out: the module cannot be found on the registry.  Do I need to do a local module load instead?

Reply:
Yes you do. I modified the pipeline so that we start testing on Terraform 1.0 for now, but since this module will work only in version 15 > I need to think in a better way to do this